### PR TITLE
S2S Migration for prod and aat from Tactical to CNP

### PR DESCRIPTION
**Pull request by bartlabno**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -62,12 +62,12 @@ task smoke(type: Test) {
...
-  setClasspath(sourceSets.e2e.runtimeClasspath)
-  include "uk/gov/hmcts/reform/sscs/functional/**"
-  exclude "uk/gov/hmcts/reform/sscs/smoke/**"
+//  group = 'Functional Tests'
+//  description = 'Executes non-destructive functional tests in AAT against a running case loader'
+//  setTestClassesDirs(sourceSets.e2e.output.classesDirs)
+//  setClasspath(sourceSets.e2e.runtimeClasspath)
+//  include "uk/gov/hmcts/reform/sscs/functional/**"
+//  exclude "uk/gov/hmcts/reform/sscs/smoke/**"
 }
```
Comment (iguannago) [inline build.gradle]: we can now put this in again.

File: infrastructure/aat.tfvars [hunk 2]
```diff
@@ -1,3 +1,4 @@
+gaps2_sftp_dir = "/incoming"
```
Comment (timja-kainos) [inline infrastructure/aat.tfvars]: is this related?

File: infrastructure/main.tf [hunk 3]
```diff
@@ -47,8 +47,11 @@ data "vault_generic_secret" "sftp_port" {
 locals {
   aseName = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
 
-  localCcdApi = "http://ccd-data-store-api-${var.env}.service.${local.aseName}.internal"
-  CcdApi = "${var.env == "preview" ? "http://ccd-data-store-api-aat.service.core-compute-aat.internal" : local.localCcdApi}"
+  local_env = "${(var.env == "preview" || var.env == "spreview") ? (var.env == "preview" ) ? "aat" : "saat" : var.env}"
+  local_ase = "${(var.env == "preview" || var.env == "spreview") ? (var.env == "preview" ) ? "core-compute-aat" : "core-compute-saat" : local.aseName}"
+
+  CcdApi = "http://ccd-data-store-api-${local.local_env}.service.${local.local_ase}.internal"
```
Comment (nitinprabhu) [inline infrastructure/main.tf]: minor one but keep the naming convention consistent `ccdApi`and not `CcdApi`?
